ui/src/components/Employee/Create/index.tsx

```diff
@@ -0,0 +1,160 @@
+  values?: CreateEmployeeFormValues;
+  isFetchingDesignations: boolean;
+  isFetchingOrgs: boolean;
+  isFetchingEmployees?: boolean;
+  designations: Designation[];
+  organizations: Organization[];
+  employee?: Employee;
+}
+
+const CreateEmployeeForm: React.FC<FormikProps<CreateEmployeeFormValues> & CreateEmployeeProps> = (
+  p,
+) => {
+  if (p.organizations.length && !p.values.organization_id) {
+    p.setFieldValue('organization_id', p.organizations[0].id);
+  }
+
+  return (
+    <div className={c('container')}>
+      <div className={c({ skeleton: p.isFetchingEmployees })}>
+        <form className={c('form')} onSubmit={p.handleSubmit}>
+          <div className={c('title-container')}>
+            <h2>{p.values ? p.values.name : 'Create Employee'} </h2>
+            <span className={c('sub-title')}>Manage your employee</span>
+          </div>
+
+          <div className={c('email-container', 'field-container')}>
+            <span className={c('field-title')}>Email</span>
+            <TextField placeholder="Enter email" type="email" name="email" />
+          </div>
+          <div className={c('name-container', 'field-container')}>
+            <span className={c('field-title')}>Name</span>
+            <TextField placeholder="Enter name" type="text" name="name" />
+          </div>
+
+          <div className={c('logo', 'field-container')}>
+            <span className={c('field-title')}>Head Shot</span>
+            <ImageUploadField className={c('image-container')} type={'file'} name="headshot" />
+          </div>
+
+          <div className={c('right-container')}>
+            <div className={c('ecode-container', 'field-container')}>
+              <span className={c('field-title')}>E-Code</span>
+              <TextField placeholder="Enter e-code" type="text" name="ecode" />
+            </div>
+
+            <div className={c('designation-container', 'field-container')}>
+              <span className={c('field-title')}>Designation</span>
+              <SelectField
+                className={c('select-field')}
+                name="designation_id"
+                options={p.designations.map((d) => ({ id: d.id, name: d.name }))}
+                isLoading={p.isFetchingDesignations}
+                isLogoHidden={true}
+              />
+            </div>
+            <div className={c('org-container', 'field-container')}>
+              <span className={c('field-title')}>Organization</span>
+              <SelectField
+                className={c('select-field')}
+                name="organization_id"
+                options={p.organizations.map((o) => ({ ...o, logo: o.square_logo }))}
+                isLoading={p.isFetchingOrgs}
+              />
+            </div>
+          </div>
+
+          <button className={c('save-button')} type="submit" disabled={p.isSubmitting}>
+            <div className={c({ 'saving-in': p.isSubmitting })}>
+              {p.isSubmitting ? 'Saving...' : 'Save'}
```

Comment:
Should we replace this with animated ellipses? Check the login form for how ellipses can be animated.